doc/man3/ERR_GET_LIB.pod

```diff
@@ -42,6 +42,13 @@ The library number, reason code, and whether the error
 is fatal, respectively.
 Starting with OpenSSL 3.0.0, the function code is always set to zero.
 
+=head1 NOTES
+
+The application developers SHOULD NOT make their decision on a particular error
+code because they are subject to change even between minor releases of OpenSSL.
+The particular error code can be considered meaningful only if it is explicitly
+documented, though new failure reasons still may appear at any time.
```

Comment:
Side note: it's also about which error in the queue they look at.  Unfortunately, some use `ERR_peek_error()` while others use `ERR_peek_last_error()`, and that usually gives different results already (and if `ERR_clear_error()` or other error queue emptying functions are never called, `ERR_peek_error()` is quite unpredictable).